docs/tools/flow-cli/flix.md

```diff
@@ -398,7 +307,7 @@ flow flix package ./multiply.template.json --lang ts --save ./multiply.ts
 
 ### Example Package Output
 ```shell
-flow flix package transfer-flow
+flow flix package https://flix.flow.com/v1/templates\?name\=transfer-flow
```

Comment:
will this also work? (without escaping characters and wrapped in quotes)
```
flow flix package "https://flix.flow.com/v1/templates?name=transfer-flow"
```

Reply:
![image](https://github.com/onflow/docs/assets/3970376/6b3c4401-06ce-418d-8684-c6027b2f94e5)

Reply:
flow cli takes it in as a string since there is no spaces, spaces are not allowed in a url. so it should be good.